src/open_inwoner/components/templates/components/Action/Actions.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n form_tags icon_tags button_tags dropdown_tags action_tags link_tags %}
 
-{% render_form form=form method="GET" spaceless=True autosubmit=True form_action=form_action %}
 <div class="actions">
+    {% render_form form=form method="GET" spaceless=True autosubmit=True form_action=form_action %}
```

Comment:
@svenvandescheur I had to move the top search-filter form-tag so it doesn't wrap the whole table and I could nest mini-forms to do the POST to the delete view, with the confirm and everything.